digital-workplace-webapps/src/main/resources/locale/navigation/portal/dw_en.properties

```diff
@@ -19,3 +19,6 @@
 portal.dw.home=Snapshot
```

Comment:
should we remove this label ?

Reply:
No it shouldn't for the existing data case